activities/member-relations/index.md

```diff
@@ -2,8 +2,10 @@
 type: Index
 ---
 
-These are all the activities, documents and templates for member relations:
+These are all the activities, documents, guides and templates for member relations:
```

Comment:
I only now see that there is no level 1 heading on this PR, optional fix

Reply:
done.